templates/profile-details.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../static/bootstrap/bootstrap.min.css">
+    <link rel="stylesheet" href="../static/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+    <script src="../static/bootstrap/bootstrap.min.js"></script>
+
+    <title>Profile Details</title>
+</head>
+<body>
+    <div style="background-color: whitesmoke;" class="container">
+        <div class="main-body">
+            <nav aria-label="breadcrumb" class="main-breadcrumb text-white">
+                <ol class="breadcrumb text-white">
+                    <li class="breadcrumb-item"><a href="index.html">Home</a></li>
+                    <li class="breadcrumb-item"><a href="javascript:void(0)">Profile</a></li>
+                    <li class="breadcrumb-item active" aria-current="page">User Profile</li>
+                </ol>
+            </nav>
+            <div class="row gutters-sm " style="background-color: orangered;">
+                <!-- START OF LEFT PART OF COLUMN -->
+                <div class="col-md-6 col-lg-5 mb-3" style="background-color: green;">
+                    <div class="card">
+                        <div class="card-body">
+                            <div class="d-flex flex-column align-items-center text-center">
+                                <img src="https://bootdey.com/img/Content/avatar/avatar7.png" alt="Admin" class="rounded-circle" width="150">
+                                <div class="mt-3 col-11">
+                                    <h4 class="">@Aykos</h4>
+                                    <h6 class="mb-4 fs-6" style="color:grey">Username</h6>
```

Comment:
Why we need this Username expression here? it is alreade known as username with "@" sign